pkg/inspector/image-inspector.go

```diff
@@ -399,9 +400,11 @@ func (i *defaultImageInspector) scanImage(s openscap.Scanner) ([]byte, []byte, e
 		return []byte(""), []byte(""), fmt.Errorf("Unable to read %s result file: %v\n", s.ScannerName(), err)
 	}
 
-	htmlScanReport, err := ioutil.ReadFile(s.HTMLResultsFileName())
-	if err != nil {
-		return []byte(""), []byte(""), fmt.Errorf("Unable to read %s HTML result file: %v\n", s.ScannerName(), err)
+	if i.opts.OpenScapHTML {
```

Comment:
In the future we should change the name of this attribute to something generic like `scanHTMLReport` to support other kinds of scanners, but the code is riddled with those so it is out of scope.